archivy/models.py

```diff
@@ -92,7 +93,15 @@ def process_bookmark_url(self):
             return None
 
         try:
-            url_request = requests.get(self.url)
+            try:
+                lc = locale.getdefaultlocale()[0].replace('_', '-')
+                if lc == "C":
+                    # fall back to en-US
+                    lc == "en-US"
+                headers = {"Accept-Language": "en-US"}
```

Comment:
I think there's a mistake here, you're not actually using the locale.

Reply:
Agreed, you are right. Thanks for the review.